include/rocksdb/utilities/backupable_db.h

```diff
@@ -25,9 +25,9 @@
 namespace ROCKSDB_NAMESPACE {
 
 // The default DB file checksum function name.
-constexpr char kDbFileChecksumFuncName[] = "FileChecksumCrc32c";
+constexpr char kDefaultDbFileChecksumFuncName[] = "FileChecksumCrc32c";
```

Comment:
Would be better to move the `kDefaultDBFileChecksumFuncName` to include/rocskdb/file_checksum.h.